### PR TITLE
Change for adaption of equations of spatial neuron except morphology

### PR DESCRIPTION
**Pull request by sparsh-989**
Below is the added functionallity
https://gist.github.com/sparsh989/41b4168241a6d8b319a51138b80e8602

### CODE REVIEW

**Comment by sparsh-989** [comment]
Screenshot 2024-08-22 at 6 42 54 PM
for the below test file
from brian2 import *
import brian2tools.mdexport
# set device 'markdown'
set_device('markdown', filename='model1_description')
defaultclock.dt = 0.01*ms

# Morphology
diameter = 1*um
length = 300*um
Cm = 1*uF/cm**2
Ri = 150*ohm*cm
N = 200
morpho = Cylinder(diameter=diameter, length=length, n=N)

# Passive channels
gL = 1e-4*siemens/cm**2
EL = -70*mV
eqs = '''
Im = gL * (EL - v) : amp/meter**2
I : amp (point current)
'''

neuron = SpatialNeuron(morphology=morpho, model=eqs, Cm=Cm, Ri=Ri,
                       method='exponential_euler')
neuron.v = EL
la = Ri * Cm**(0.5)  

print("Electrotonic length: %s" % la)

neuron.I[0] = 0.02*nA # injecting at the left end
run(100*ms, report='text')

File: brian2tools/baseexport/collector.py [hunk 1]
```diff
@@ -94,6 +94,96 @@ def collect_NeuronGroup(group, run_namespace):
 
     return neuron_dict
 
+def collect_SpatialNeuron(group, run_namespace):
+
+    """
+    Collect information from `brian2.groups.neurongroup.NeuronGroup`
```
Comment (mstimberg) [inline brian2tools/baseexport/collector.py:100]: Please refer to `SpatialNeuron` here

File: brian2tools/baseexport/collector.py [hunk 2]
```diff
@@ -94,6 +94,96 @@ def collect_NeuronGroup(group, run_namespace):
...
+    and return them in a dictionary format
+
+    Parameters
+    ----------
+    group : brian2.groups.neurongroup.NeuronGroup
```
Comment (mstimberg) [inline brian2tools/baseexport/collector.py:105]: `SpatialNeuron` here and below as well

File: brian2tools/baseexport/collector.py [hunk 3]
```diff
@@ -94,6 +94,96 @@ def collect_NeuronGroup(group, run_namespace):
...
+
+    run_namespace : dict
+        Namespace dictionary
+
+    Returns
+    -------
+    neuron_dict : dict
+        Dictionary with extracted information
+    """
+    print("hello2")
```
Comment (mstimberg) [inline brian2tools/baseexport/collector.py:116]: Please remove

**Review by mstimberg — CHANGES_REQUESTED** [review]

**Review by mstimberg — CHANGES_REQUESTED** [review]
I did not test things in detail, but from a basic check this all looks good! I left a few minor comments

**Review by mstimberg — APPROVED** [review]

**Pull request merged**